unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/WearablesHelper/WearablesFetchingHelper.cs

```diff
@@ -13,7 +14,7 @@ public static class WearablesFetchingHelper
         // TODO: change fetching logic to allow for auto-pagination
         // The https://nft-api.decentraland.org/v1/ endpoint doesn't fetch L1 wearables right now, if those need to be re-converted we should use that old endpoint again and change the WearablesAPIData structure again for that response.
         public const string BASE_FETCH_URL = "https://peer.decentraland.org/lambdas/collections";
-        public const string COLLECTIONS_FETCH_PARAMS = "?sortBy=newest&first=1000"; 
+        public const string COLLECTIONS_FETCH_PARAMS = "?sortBy=newest&first=1000";
```

Comment:
Could be nice to split the first=1000 parameter to create a constant for the maximum number to help possible future change of this number, but it's just my preference